tests/js/spec/views/eventsV2/table/cellAction.spec.jsx

```diff
@@ -120,5 +125,69 @@ describe('Discover -> CellAction', function() {
         }),
       });
     });
+
+    it('go to summary button goes to transaction summary page', function() {
+      wrapper.find('button[data-test-id="transaction-summary"]').simulate('click');
```

Comment:
Do you not need to open the menu first?

Reply:
There's a `beforeEach()` that already does this.